frontend/src/data_context_global.tsx

```diff
@@ -121,6 +121,20 @@ export interface Breeding {
   litter_size: number | null;
 }
 
+export interface ExtendedBreeding {
```

Comment:
Should probably extend `Breeding`? (https://www.typescriptlang.org/docs/handbook/2/objects.html#extending-types)

Reply:
Yes, indeed. ExtendedBreeding should extend breeding. I guess I had a thought when writing this and then didn't fully think it to the end :)